lib/hammer_cli_foreman_tasks/task_progress.rb

```diff
@@ -40,7 +44,7 @@ def progress_message
 
     def render_result
       puts @task['humanized']['output'] unless @task['humanized']['output'].to_s.empty?
-      STDERR.puts @task['humanized']['errors'].join("\n") unless @task['humanized']['errors'].to_s.empty?
+      STDERR.puts "Error: #{@task['humanized']['errors'].join("\n")}" unless @task['humanized']['errors'].to_s.empty?
```

Comment:
Remove the `to_s` here. `errors` is an array, which even if empty will result in a non-empty string from `to_s`.

```
[1] pry(main)> arr = []
[]
[2] pry(main)> arr.to_s
"[]"
[3] pry(main)> arr.to_s.empty?
false
```